docs/containers-storage.conf.5.md

```diff
@@ -34,6 +34,20 @@ The `storage` table supports the following options:
   container storage graph dir (default: "/var/lib/containers/storage")
   Default directory to store all writable content created by container storage programs.
 
+**rootless_storage_path**="$HOME/.local/share/containers/storage"
```

Comment:
should it be specific to rootless or can we name it `user_storage_path` so that root containers also look it up if `GraphRoot` is not specified?

Reply:
I would think the graphroot would just get default if not specified, rather then looking for this path.